options/locale/locale_en-US.ini

```diff
@@ -899,8 +899,7 @@ migrate.clone_address_desc = The HTTP(S) or Git 'clone' URL of an existing repos
 migrate.github_token_desc = You can put one or more tokens with comma separated here to make migrating faster because of Github API rate limit. WARN: Abusing this feature may violate the service provider's policy and lead to account blocking.
 migrate.clone_local_path = or a local server path
 migrate.permission_denied = You are not allowed to import local repositories.
-migrate.permission_denied_blocked = You are not allowed to import from blocked hosts.
-migrate.permission_denied_private_ip = You are not allowed to import from private IPs.
+migrate.permission_denied_blocked = You can not import from disallowed hosts, please check your ALLOWED_DOMAINS/ALLOW_LOCALNETWORKS/BLOCKED_DOMAINS settings.
```

Comment:
I had a quick look, it seems like that normal users can also get this error - I don't think it makes sense that a normal user should check ` ALLOWED_DOMAINS/ALLOW_LOCALNETWORKS/BLOCKED_DOMAINS` settings as that's only appropriate for the administrator.

Reply:
Then what about "please ask the admin to check  ALLOWED_DOMAINS/ALLOW_LOCALNETWORKS/BLOCKED_DOMAINS settings"?

I want to expose the details here, otherwise many junior admins will open issues in github ......

Reply:
> I want to expose the details here, otherwise many junior admins will open issues in github ......

I get that.



> Then what about "please ask the admin to check ALLOWED_DOMAINS/ALLOW_LOCALNETWORKS/BLOCKED_DOMAINS settings"?

That's possible, another form could be "You cannot import from disallowed hosts that the server administrator has configured".